libs/vault/src/cipher-form/components/item-details/item-details-section.component.ts

```diff
@@ -91,7 +91,7 @@ export class ItemDetailsSectionComponent implements OnInit {
   }
 
   get organizations(): Organization[] {
-    return this.config.organizations;
+    return this.config.organizations.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
🎨 We should use the sort function provided by the `i18nService` and our util helper. 
```suggestion
    return this.config.organizations.sort(Utils.getSortFunction(this.i18nService, "name"));
```